src/OpenLoco/src/Vehicles/Vehicle.h

```diff
@@ -673,6 +675,7 @@ namespace OpenLoco::Vehicles
         {
             return (breakdownFlags & flagsToTest) != BreakdownFlags::none;
         }
+        VehicleBogie* flipCar();
```

Comment:
From the way this function works i think it should be a free function rather than a member function. In general we are trying to move to more free functions and use members less.